app/ui/src/app/connections/list/list.component.html

```diff
@@ -108,24 +95,14 @@ <h1>Register Applications</h1>
         </div>
       </ng-container>
       <ng-container *ngIf="showNewConnection && connections.length > 0">
-        <div class="connection new-connection col-xs-12 col-sm-3 col-md-2 card"
-             (click)="onSelect(undefined, $event)">
-          <div class="card-pf card-pf-view card-pf-view-select card-pf-view-single-select">
-            <div class="card-pf-body">
-
-              <!-- Card Icon -->
-              <div class="card-pf-top-element">
-                <span class="card-pf-icon-large pficon pficon-add-circle-o">
-                </span>
-              </div>
-
-              <!-- Card Description / Item Overview -->
-              <div class="card-pf-info text-center">
-                <button class="btn btn-default" (click)="onSelect(undefined, $event)">Create Connection</button>
-              </div>
-            </div>
+        <syndesis-empty-state-card class="col-md-6 col-lg-3" (click)="onSelect(undefined, $event)">
+          <div class="empty-state-card__header">
+            <span class="card-pf-icon-large pficon pficon-add-circle-o"></span>
           </div>
-        </div>
+          <div class="empty-state-card__body">
+            <button class="btn btn-default">Create Connection</button>
+          </div>
+        </syndesis-empty-state-card>
       </ng-container>
```

Comment:
You can spare the `<ng-container>` element here and apply the `*ngIf` directly on the `<syndesis-empty-state-card>` element, making the code more readable.

For the record, 50% of `<ng-content>` tags found in the code throughout the application are not necessary and hence redundant.

Reply:
I recall @sjcox-rh mentioned that this big plus icon should be always displayed in the empty card, so it's redundant and it should become part of `empty-state-card.component.ts` instead.

I'll post this as a comment, since I see 2 alternatives here:

- Correct this and send a final commit (preferred)
- Merge this and iterate on it again in the future (might never happen LOL)

Pick one ;)

Reply:
If you go for the latter, feel free to dismiss my review and let's get this merged straight away

Reply:
thanks @deelman, corrected and updated the branch.